docs/recipes/custom-response-composition.mdx

```diff
@@ -39,7 +39,7 @@ export const handlers = [
   rest.get('/user', (req, res, ctx) => {
     // This mocked response has realistic
     // response time automatically.
-    return delayedResponse(
+    return delayedResponse(() =>
```

Comment:
Is there an issue you're experience regarding this?

`createResponseComposition` returns what is a `res` function, just as the resolver gives it to you in the args. I think it should accept utilities like `ctx.json()` straight away, because the composition function creates that response object internally, and then maps it through each utility:

https://github.com/mswjs/msw/blob/1caf8df3910de4bdbdecbe4faa4f2778db55e141/src/response.ts#L58-L62

So the `delayResponse` signature is

```ts
const delayResponse = (...transformers: ResponseTransformer[]) => MockedResponse
```

which is correctly reflected in this usage example. 

Am I missing something?

Reply:
I copied the entire example verbatim into my project, and it didn't work (I got an error message from `Array.reduce` about not being able to invoke something). I changed it to a function returning a response, and it did work.

I assumed it was just a typo in the docs, but if not, there might be something "wrong" with my code that happens to make this work anyway.

If the docs are actually as intended, feel free to ignore. It just didn't work for me, but that's easily ascribable to other errors. I didn't try to create a condensed reproduction of the "bug" or anything.

Reply:
Can you double-check the version of `msw` you're using? 

I don't recall us changing anything in this logic but just to be safe. I believe the docs are written as intended. Please, if you can reliably reproduce this, would you mind [opening an issue](https://github.com/mswjs/msw/issues/new?assignees=&labels=bug%2Cscope%3Anode%2Cneeds%3Atriage&template=02-issue-nodejs.yml)?